docs/docs/cmd/teams/team/team-get.md

```diff
@@ -31,3 +31,81 @@ Get information about Microsoft Teams team with name _Team Name_
 ```sh
 m365 teams team get --name "Team Name"
 ```
+
+## Response
+
+=== "JSON"
+
+    ``` json
+    {
+      "id": "a40210cd-0060-4b91-aaa1-a44e0853d979",
+      "createdDateTime": "2022-10-31T12:50:42.819Z",
+      "displayName": "Architecture",
```

Comment:
Let's try to align the output with the data used in the example. Same goes for output types text and csv.